xformers/components/attention/csrc/cuda/mem_eff_attention/attention_backward_generic.cu

```diff
@@ -121,90 +87,99 @@ mem_efficient_attention_backward_generic(
   }
 
   DISPATCH_MAXK(([&] {
-    DISPATCH_TYPES(([&]() {
```

Comment:
Similarly here, this is just a difference in indentation right?

Reply:
Yes. I want to change this in the future and have a generic `DISPATCH_KERNEL(...)` that does all the dispatching, to avoid this kind of diffs, and all the identation